### ISSUE TITLE
How to get candidates in training data of intrinsic?

### ISSUE THREAD

**Issue by SatoMichi** [CogComp/TacoLM#3]
Hello,

I am trying to predict different text data with the model, and find out some features needed for the model are unclear for me. 

In the data/intrinsic, there is 4 training (evaluation) file for the model.  Each line are in the format "sentence with MASK", "target id", "gold label", "candidates", "dim".

My understanding from observing the code is, "target id" is an index that points to the "MASK" of the sentence, "gold label" is the correct label of the data. However, I could not understand what "candidates" are representing. 

For the prediction task, it looks like prediction are done in following way.
1. Using (input_ids, segment_ids, input_mask) generated from "sentence" and, target_ids to get "cls" vector.
2. reshape "cls" vector with cls.view(-1, 30522) which is called logits
3. using logits and "candidates" to compute scores_in_order
4. using scores_in_order to predict the predicted_relative_label_id which should be the model's prediction.

Since I want to get the predicted_relative_label_id as a prediction, I will need to get candidates, but I do not know how to get it.

What are candidates and how to get it from the text?
